R/codemeta_readme.R

```diff
@@ -107,23 +107,22 @@ guess_readme_path <- function(root) {
     return(NULL)
   }
 
+  # Filter for Rmd files
   readme_rmd <- grep("\\.[Rr]md$", readmes, value = TRUE)
 
-  if (length(readme_rmd) == 0) {
+  # If there is a Rmd file, use this one, else filter for md files
+  readme_file <- if (length(readme_rmd)) {
 
```

Comment:
could you put the assignment inside the if and else? I personally find it easier to read :-)